fingerprint2.js

```diff
@@ -428,6 +460,23 @@
         done(keys);
       }, 1);
     },
+    mediaDevicesKey: function(keys, done) {
+        //Adds the device key for each media device to the devicesArr
+        if(navigator.mediaDevices && !this.options.excludeMediaDevices){
```

Comment:
option check should come first, because we don't want to touch mediaDevices if user has them excluded